recipes/geoh5py/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "geoh5py" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MiraGeoscience/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 71ef863971355dcaf255c5e1824be5cf091c87940ab930001ab08ca8bfd11441
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<3.11
+    - poetry-core >=1.0.0
+    - setuptools
+    - pip
+  run:
+    - python >=3.9.0,<4.0.0,<3.11
+    - h5py >=3.2.1,<4.0.0
+    - numpy >=1.23.5,<1.24.0
+    - pillow >=10.3.0,<10.4.0
+
+test:
+  imports:
+    - geoh5py
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.mirageoscience.com/mining-industry-software/python-integration/
+  summary: 'Python API for geoh5, an open file format for geoscientific data.'
+  description: |
+    The geoh5py library has been created for the manipulation and storage of a wide range of 
+    geoscientific data (points, curve, surface, 2D and 3D grids) in GEOH5 file format. Users will 
+    be able to directly leverage the powerful visualization capabilities of 
+    (Geoscience ANALYST)<https://mirageoscience.com/mining-industry-software/geoscience-analyst/> 
+    along with open-source code from the Python ecosystem.
+  license: LGPL-3.0-only
+  license_file:
+    - COPYING
+    - COPYING.LESSER
+  doc_url: https://mirageoscience-geoh5py.readthedocs-hosted.com/
+  dev_url: https://github.com/MiraGeoscience/geoh5py
+
+extra:
+  recipe-maintainers:
+    - SophieCurinier
+    - sebhmg
+  feedstock-name: MiraGeoscience
```

Comment:
The feedstock, as far as I know, only for existing packages that were renamed and/or have multiple outputs. That is not the case here and this can break the package publication b/c there isn't such a name. The feedstock name will be `geoh5py-feedstock` when we merge this.

Reply:
Hi @ocefpaf ,

We thought we could have 1 single feedstock with all our packages inside (geoh5py, mira-simpeg, etc.). We hadn't seen that the feedstock-name only concerned renamed packages. I'm going to delete this line, so the feedstock will be `geoh5py-feedstcok`.

Thank you for your help

Reply:
conda-forge's model is 1 package (or multiple outputs from the same package) per feedstock (repository). We di not aggregate them.